admin-dev/themes/default/template/controllers/referrers/calendar.tpl

```diff
@@ -35,11 +35,13 @@
 					<input type="submit" name="submitDateYearPrev" class="btn btn-default submitDateYearPrev" value="{$translations.Year}-1" />
 					<p>
 						<span>{if isset($translations.From)}{$translations.From}{else}{l s='From:' d='Admin.Global'}{/if}</span>
-						<input type="text" name="datepickerFrom" id="datepickerFrom" value="{$datepickerFrom|escape}" class="datepicker" />
+            <input type="text" class="datepicker" data-target="#datepickerFrom"/>
+            <input type="hidden" name="datepickerFrom" id="datepickerFrom" value="{$datepickerFrom|escape}" />
 					</p>
 					<p>
 						<span>{if isset($translations.To)}{$translations.To}{else}<span>{l s='To:' d='Admin.Global'}</span>{/if}</span>
-						<input type="text" name="datepickerTo" id="datepickerTo" value="{$datepickerTo|escape}" class="datepicker" />
+            <input type="text" class="datepicker" data-target="#datepickerTo"/>
+						<input type="hidden" name="datepickerTo" id="datepickerTo" value="{$datepickerTo|escape}" />
```

Comment:
Why do we need 2 inputs there?

Reply:
Because with the datepicker, we need the formatted date and the input date (MYSQL format)

Reply:
Ok 👍